docs/product-user-guide/src/main/asciidoc/decision-tables-rule-table-entries-ref.adoc

```diff
@@ -0,0 +1,85 @@
++
+If this cell is empty, the code snippet in the cell below it must result in a valid conditional element on its own. For example, instead of `Person` as the object type and `age` as the constraint (separate cells), you can enter `Person( age == $1)` in the object type cell and leave the constraint cell below it empty.
++
+To include a pattern without constraints, you can write the pattern in front of the text of another pattern, with or without an empty pair of parentheses. You can also append a `from` clause to the pattern.
++
+If the pattern ends with `eval`, code snippets produce boolean expressions for inclusion into a pair of parentheses after `eval`.
+* *Second cell:* Text in the second cell below `CONDITION` is processed as a constraint on the object reference in the first cell. The code snippet in this cell is modified by interpolating values from cells farther down in the column. If you want to create a constraint consisting of a comparison using `==` with the value from the cells below, then the field selector alone is sufficient. Any other comparison operator must be specified as the last item within the snippet, and the value from the cells below is appended. For all other constraint forms, you must mark the position for including the contents of a cell with the symbol `$param`. Multiple insertions are possible if you use the symbols `$1`, `$2`, etc., and a comma-separated list of values in the cells below. However, do not separate `$1`, `$2`, etc., by commas, or the table will fail to process.
++
+To expand a text according to the pattern `forall(_DELIMITER_){_SNIPPET_}`, repeat the `_SNIPPET_` once for each of the values of the comma-separated list in each of the cells below, insert the value in place of the symbol `$`, and join these expansions by the given `_DELIMITER_`. Note that the `forall` construct may be surrounded by other text.
++
+If the first cell contains an object, the completed code snippet is added to the conditional element from that cell. A pair of parentheses is provided automatically, as well as a separating comma if multiple constraints are added to a pattern in a merged cell. If the first cell is empty, the code snippet in this cell must result in a valid conditional element on its own. For example, instead of `Person` as the object type and `age` as the constraint (separate cells), you can enter `Person( age == $1)` in the constraint cell and leave the cell above it empty.
+* *Third cell:* Text in the third cell below `CONDITION` is a descriptive label that you define for the column, as a visual aid.
+* *Fourth cell:* From the fourth row on, non-blank entries provide data for interpolation. A blank cell omits the condition or constraint for this rule.
+
+Actions::
+
+For columns headed `ACTION`, the cells in successive lines result in an action statement:
```

Comment:
"successive" is another that I pause on for translation potential issues. Would "consecutive" be better?

Reply:
Sure.